retool/README.md

```diff
@@ -0,0 +1,25 @@
+# Retool
+
+## Overview
+Monitoring and analytics offer mission-critical insights, but developers often have to jump between separate, siloed, and often custom tools to take action on the insights—leading to inefficient or ineffective responses to those insights. 
+
+Retool helps developers create custom apps that embed directly into a Datadog dashboard, giving you the ability to take action and automate workflows without having to leave Datadog. 
+
+
+### Metrics
+Datadog’s embedded Retool app for Elasticsearch Management combines existing visibility into key Elasticsearch metrics and logs with the power to manage clusters, accounts, and more without leaving your Datadog dashboard.
+
+### Dashboards
+Retool built an embedded app for Elasticsearch Management. You can already monitor Elasticsearch metrics, traces, and logs in Datadog today. With our embedded app, developers can take action on their rich Datadog insights directly in the Datadog dashboard. These actions include:
+
+- Add a new index with shards and replicas
+- Manage nodes by rerouting shards and excluding indexes
+- Create new snapshots and restore indexes
+
+## Setup
+You can set this up as your default dashboard by cloning the dashboard and then clicking the iframe to sign up for Retool. You'll be prompted to connect to your ElasticSearch cluster and in less than 5 clicks you'll be up and running with a fully functioning app.
```

Comment:
It would be good to add additional context here. Where are you cloning the dashboard? (I know you mean in Datadog but a user might not). Is there anything you need to have ready to connect your ElasticSearch cluster? What are the 5 clicks?

Reply:
@jamiecuffe provided updated copy